Gruntfile.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(grunt) {
+    var bowerPrefix = 'bower_components/bower_components/'
+
+    var vendorB2 = [
+        bowerPrefix + 'jquery-1.7.1-legacy/jquery.js',
+        bowerPrefix + 'knockout-2.3.0-legacy/knockout.js',
+        bowerPrefix + 'underscore-legacy/underscore-min.js',
+    ];
+    var vendorB3 = [
+        bowerPrefix + 'jquery/dist/jquery.min.js',
+        bowerPrefix + 'jquery-form/jquery.form.js',
+        bowerPrefix + 'underscore/underscore.js',
+        bowerPrefix + 'jquery.cookie/jquery.cookie.js',
+    ];
+
+    grunt.initConfig({
+        jasmine: {
+            app_manager: {
```

Comment:
this is pretty unique from a lot of test runners that i've been looking into. we can define a completely different set of options for jasmine and run them all in one go. this example shows an app_manager test suite that includes the B2 vendor. but if i had tests that relied on B3 vendor i could easily make another key that is `app_manager_b3` and define the options.

the downside to this is it's a bit clunky to run the tests in the browser. you can specify an `outfile` which will tell grunt where to build the jasmine runner file. it defaults `_SpecRunner.html`. so in order to run the tests in the browser you have to do `jasmine:app_manager:build`. i'm thinking we can customize the `outfile` based on the app